### PR TITLE
feat: beautiful code markdown

### PR DESCRIPTION
**Pull request by juliahermak**
Before:
Screenshot 2023-10-09 22 15 51
After:
Screenshot 2023-10-09 22 16 04

### CODE REVIEW

**Comment by bludnic** [comment]
[Feat: Beautiful code markdown](https://trello.com/c/b5jkzwzx/381-feat-beautiful-code-markdown)

File: src/assets/styles/components/_chat.scss [hunk 1]
```diff
@@ -537,10 +534,22 @@ $scroll-bar-width: 4px;
       a {
         color: map-get($blue, 'lighten-2');
       }
+
+      code{
+        color: #ffffff;
```
Comment (bludnic) [inline src/assets/styles/components/_chat.scss:539]:
```suggestion
        color: map-get($shades, 'white');
```

File: src/assets/styles/components/_chat.scss [hunk 2]
```diff
@@ -537,10 +534,22 @@ $scroll-bar-width: 4px;
...
+        border-color: map-get($grey, 'darken-2');
+        background-color: map-get($adm-colors, 'secondary2-slightly-transparent');
+      }
       pre {
-        background-color: map-get($grey, 'darken-4');
+        
         color: inherit;
         border-color: map-get($grey, 'darken-2');
+        background-color: map-get($adm-colors, 'secondary2-slightly-transparent') !important;
```
Comment (bludnic) [inline src/assets/styles/components/_chat.scss:547]: will it work without `!important`?

**Comment by juliahermak** [comment]
Before:
Screenshot 2023-10-11 04 23 29
After:
Screenshot 2023-10-11 04 23 59

File: src/assets/styles/components/_chat.scss [hunk 3]
```diff
@@ -236,15 +231,12 @@ $scroll-bar-width: 4px;
...
-      border-width: 1px;
+      border-radius: 6px;
       overflow-x: auto;
       max-width: 100%;
-      border-color: rgb(224, 224, 224);
-
-      code:before {
-        content: '';
+      
+      code{
```
Comment (bludnic) [inline src/assets/styles/components/_chat.scss:238]:
```suggestion
      code {
```

run prettier

**Review by bludnic — CHANGES_REQUESTED** [review]

**Comment by juliahermak** [comment]
I added all the fonts from this list and added one more at Bludnic request.
Screenshot 2023-10-15 11 07 18

**Comment by bludnic** [comment]
@martiliones see discussion in trello card. This select was added for testing purposes.

**Review by bludnic — APPROVED** [review]

**Pull request merged**